File: BmiCalc/app/src/main/java/pamo/bmicalc/ui/quiz/QuizAnswer.java
package pamo.bmicalc.ui.quiz;

public class QuizAnswer {
    private int answer_id;
    private String answer_text;
    private boolean answer_valid;

    public QuizAnswer(int answer_id, String answer_text, boolean answer_valid) {
        this.answer_id = answer_id;
        this.answer_text = answer_text;
        this.answer_valid = answer_valid;
    }

    public String getAnswer_text() {
        return answer_text;
    }

    public boolean isAnswer_valid() {
        return answer_valid;
    }
}

File: BmiCalcKotlin/app/src/main/java/pamo/bmicalc_kotlin/ui/quiz/QuizAnswer.kt
package pamo.bmicalc_kotlin.ui.quiz

class QuizAnswer(answerId: Int, answerText: String, answerValid: Boolean) {
    private var answerId: Int? = null
    private var answerText: String = ""
    private var answerValid: Boolean = false

    init {
        this.answerId = answerId
        this.answerText = answerText
        this.answerValid = answerValid
    }

    fun getAnswerText(): String {
        return this.answerText
    }

    fun isAnswerValid(): Boolean {
        return this.answerValid
    }
}